www/nginx/src/opnsense/mvc/app/models/OPNsense/Nginx/Nginx.xml

```diff
@@ -773,6 +773,27 @@
         <default>0</default>
         <Required>Y</Required>
       </https_only>
+      <ssl_protocols type="OptionField">
+        <multiple>Y</multiple>
+        <Sorted>Y</Sorted>
+        <OptionValues>
+          <TLSv1 value="TLSv1">TLSv1</TLSv1>
+          <TLSv1_1 value="TLSv1.1">TLSv1.1</TLSv1_1>
```

Comment:
Remove 1.1 and 1. Both are considered insecure and should not be used anymore (deprecated).
They are also not supported anymore by browsers.

See: https://en.wikipedia.org/wiki/Transport_Layer_Security#History_and_development

Reply:
I would not remove these options, in real world there are still Windows XPs around (like ATM), there are old printers or Handhelds where nginx then would be a no-go

Reply:
@mimugmail thanks!
@fabianfrz also think that we should keep them. since they are still supported by openssl, the user should be able to do so (old clients, old servers etc). May be I can add a warning text about TLS1/1.1 deprecation?

Reply:
Such devices that do not support TLS 1.2 are EoL since probably > 10yrs (2008 is the release of TLS 1.2) and TLS 1.1 is going to be disabled in most public web server configurations anyway. So this machines will be cut off the web anyway soon.

I do not care about unsupported operating systems that just exist because some admins don't care about security. Windows XP machines should be disconnected from any network or in a dedicated completely isolated network since they do not get any patches so malware may infect them easily. We know how those stories end: They complain that they became the victim of ransomware.

Current browsers do end the support for TLS 1.1 as well BTW. There is no good reason to support something older than 1.2.

Reply:
@fabianfrz
I cannot blame the admins who work with the equipment that is, and not with the one that they would like)
OK. When really needed, the use of depricated protocols is still possible via hook.
Removed from the GUI